### ISSUE TITLE
Croupier: Allow Game Restart

### ISSUE THREAD

**Issue by afandria** [vanadium/issues#979]
After #978, allow users to quit and restart game.

This involves
- storing the last game id and syncgroup name
- changing the UI to show a Resume Game button that automatically jumps to the latest game stored.
- allowing users to quit in-game

Potentially allow users to resume more games later (by browsing a list of the games they have played in the past).

**Comment by afandria** [comment]
This is done now. Resuming games (besides the last) is not a priority feature.
